mongolog/management/commands/mongolog_purge.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is the main management command that we will use to purge mongolog records from out db.